<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ZHWDialog</class>
 <widget class="QDialog" name="ZHWDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>251</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>HW device properties</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>15</number>
       </property>
       <item>
        <widget class="QCheckBox" name="alMMap" >
         <property name="minimumSize" >
          <size>
           <width>170</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>MMap emulation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alSyncPtr" >
         <property name="text" >
          <string>Use SYNC_PTR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alNonblock" >
         <property name="text" >
          <string>Nonblocking mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Card</string>
         </property>
         <property name="buddy" >
          <cstring>alCard</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alCard" >
         <property name="minimumSize" >
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Device</string>
         </property>
         <property name="buddy" >
          <cstring>alDevice</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alDevice" >
         <property name="minimumSize" >
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Subdevice</string>
         </property>
         <property name="buddy" >
          <cstring>alSubdevice</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alSubdevice" >
         <property name="minimumSize" >
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Format</string>
         </property>
         <property name="buddy" >
          <cstring>alFormat</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alFormat" >
         <property name="minimumSize" >
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Channels</string>
         </property>
         <property name="buddy" >
          <cstring>alChannels</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alChannels" >
         <property name="minimumSize" >
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Rate</string>
         </property>
         <property name="buddy" >
          <cstring>alRate</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alRate" >
         <property name="minimumSize" >
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="9" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>alCard</tabstop>
  <tabstop>alDevice</tabstop>
  <tabstop>alSubdevice</tabstop>
  <tabstop>alFormat</tabstop>
  <tabstop>alChannels</tabstop>
  <tabstop>alRate</tabstop>
  <tabstop>alMMap</tabstop>
  <tabstop>alSyncPtr</tabstop>
  <tabstop>alNonblock</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <includes/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ZHWDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>388</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ZHWDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>481</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
